src/components/user/event.tsx

```diff
@@ -18,6 +18,7 @@ import {
   SetThreadStateNotificationEvent,
   SetUuidStateNotificationEvent,
   TaxonomyTerm,
+  TaxonomyTermType,
   AbstractUuid,
   Thread,
 } from '@serlo/api'
```

Comment:
internal note: maybe using the new graphql codegen here is a good refactoring because the query is already quite complex and it could help with the types